docs/source/examples.rst

```diff
@@ -12,7 +12,7 @@ Water MP2 Energy
 .. code:: python
 
   {
-    "schema_name": "QC_JSON",
+    "schema_name": "qc_schema_output",
     "schema_version": 0,
```

Comment:
Can you bump this too `"schema_version": 1`? `QC_JSON` is correct for version 0 I believe.

Reply:
Done.